src/components/featured-item/featured-item.scss

```diff
@@ -139,7 +139,7 @@ $featured-item: null !default;
     }
 
     .ecl-featured-item__description {
-      margin: map.get($featured-item, 'description-spacing-right');
+      margin-left: 0;
```

Comment:
We support RTL display for the components, so we don't use left right top and bottom for spacings like paddings and margins, so this should be margin-inline-start.
To test it, you have a checkbox in the Css resources panel that enables the RTL css

Reply:
thanks, the margins were wrong also on rtl a lot... i compared with what we have on v4 prod and it's the same now